library/systemd/src/lib/yast2/systemd_unit.rb

```diff
@@ -222,7 +264,7 @@ def load_systemd_properties
     class InstallationProperties < OpenStruct
       include Yast::Logger
 
-      def initialize(systemd_unit)
+      def initialize(systemd_unit, _property_text)
```

Comment:
so if I get it, it is ignored in installation and always readed even when cached property text is passed. It is a bit surprising behavior.

Reply:
Damn, it seems like I missed this and this will break during the installation :(

Reply:
Fixed